packages/tpetra/core/src/Tpetra_BlockMultiVector_decl.hpp

```diff
@@ -320,15 +320,25 @@ class BlockMultiVector :
   /// This is a class ("static") method so that you can make and reuse
   /// a point Map for creating different BlockMultiVector instances,
   /// using the more efficient four-argument constructor.
+  ///
   static map_type
   makePointMap (const map_type& meshMap, const LO blockSize);
 
+  /// \brief Create and return the point Map corresponding to the
+  ///   given mesh Map and block size.
+  ///
+  /// This is a class ("static") method so that you can make and reuse
+  /// a point Map for creating different BlockMultiVector instances,
+  /// using the more efficient four-argument constructor.
+  static Teuchos::RCP<const map_type>
+  makePointMapRCP (const map_type& meshMap, const LO blockSize);
+
   /// \brief Get this BlockMultiVector's (previously computed) point Map.
   ///
   /// It is always valid to call this method.  A BlockMultiVector
   /// always has a point Map.  We do not compute the point Map lazily.
-  map_type getPointMap () const {
-    return pointMap_;
+  const map_type getPointMap () const {
```

Comment:
If there's a motivation for returning `const` here, let's explain in the function docs

Reply:
Then I think we want to reuturn `Teuchos::RCP<const map_type>`?

Reply:
Ah I follow now